op-node/rollup/engine/engine_controller.go

```diff
@@ -512,6 +517,7 @@ func (e *EngineController) insertUnsafePayload(ctx context.Context, envelope *et
 	if err != nil {
 		return derive.NewTemporaryError(fmt.Errorf("failed to update insert payload: %w", err))
 	}
+	e.log.Debug("engine.NewPayload returned", "ref", ref, "status", status.Status)
```

Comment:
Would be nice to add that this log is triggered inside the `insertUnsafePayload`.